config/routes.rb

```diff
@@ -1,6 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  post '/users/sign-up', to: 'users#sign_up'
```

Comment:
로그인까지 구현하실꺼면 users 를 기준으로 nested 구조로 선언하는게 나을 것 같습니다. 

```suggestion
  namespace :users do
    post :sign_up, path: 'sign-up'
  end
```
그리고 터미널에서 rails routes 사용하면 구현된 라우트 패스들 조회할 수 있으니 잘 활용하시면 좋을 것 같습니다. 

rails에서는 다양한 라우트 패스 선언법을 지원하는데 너무 방대해서 혼란이 있을 수도 있을거 같아 이전에 일승님이 지식 공유 섹션에서 잘 설명해준 자료도 공유드립니다.

https://dramancompany.atlassian.net/wiki/spaces/API/pages/27818098722/2021

Reply:
해당 내용을 수정하여 반영하였습니다.